sr_data_visualization/src/sr_data_visualization/sr_data_visualizer_gui.py

```diff
@@ -92,10 +94,29 @@ def fill_layout(self):
             self.layout.addWidget(QLabel("No hand connected or ROS bag is not playing"), alignment=Qt.AlignCenter)
             return
 
+        self.hand_labels = []
+        self.hand_id = 0
```

Comment:
hand_id within our codebase is usually defined as lh or rh, try keeping this variable only with these values